src/localizers/globalize.js

```diff
@@ -10,6 +10,12 @@ let timeRangeFormat = ({ start, end }, culture, local)=>
   local.format(start, { time: 'short' }, culture) +
     ' — ' + local.format(end, { time: 'short' }, culture)
 
+let timeRangeStartFormat = ({ start, end }, culture, local)=>
+  local.format(start, 't', culture) + ' — '
```

Comment:
I would mirror what the complete format above does in this case: `{ time: 'short' }`